src/app/games/six/SixTutorial.ts

```diff
@@ -0,0 +1,157 @@
+import { SixGameState } from 'src/app/games/six/SixGameState';
+import { SixMove } from 'src/app/games/six/SixMove';
+import { Coord } from 'src/app/jscaip/Coord';
+import { Player } from 'src/app/jscaip/Player';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+import { DidacticialStep } from '../../components/wrapper-components/didacticial-game-wrapper/DidacticialStep';
+
+const _: number = Player.NONE.value;
+const O: number = Player.ZERO.value;
+const X: number = Player.ONE.value;
+
+export class SixTutorialMessages {
+
+    public static readonly MOVEMENT_NOT_DISCONNECTING: string = $localize`This move is not disconnecting a piece of your opponent. Try again with another piece.`;
+
+    public static readonly MOVEMENT_SELF_DISCONNECTING: string = $localize`You lost one of your pieces during this move. There is a way to disconnect an opponent's piece without losing it, try again!`;
+}
```

Comment:
J'ai changé la phrase en 
**Vous avez perdu une de vos pièce pendant ce mouvement, il y a un moyen de déconnecter une pièce adversaire sans perde aucunes pièces, recommencez!**

Si tu peux adapter la tienne en 
**You lost one of your pieces during this move. There is a way to disconnect an opponent's piece without losing any of yours, try again!**